src/Core/Scheduler/Providers/WFS_Provider.js

```diff
@@ -98,7 +98,11 @@ WFS_Provider.prototype.getFeatures = function getFeatures(crs, tile, layer) {
     if (result.feature !== undefined) {
         return Promise.resolve(result);
     }
-    return Fetcher.json(url, layer.networkOptions).then(geojson => assignLayer(Feature2Mesh.convert(GeoJSON2Features.parse(crs, geojson, tile.extent)), layer));
+
+    const defaultCallbacks = { onMeshCreated: undefined, color: undefined, altitude: undefined, extrude: undefined };
```

Comment:
You can remove `defaultCallbacks`. And modify ` Feature2Mesh.convert` signature to be: `Feature2Mesh.convert(options = {})`